code/nodes/runModels/funcs.q

```diff
@@ -17,35 +20,37 @@ runModels.jsonParse:{[scoreFunc]
 
 // @kind function
 // @category runModels
-// @fileoverview Set value of random seed for reproducibility
-// @param config {dict} Information relating to the current run of AutoML
-// @return {Null} Value of seed is set
+// @desc Set value of random seed for reproducibility
+// @param config {dictionary} Information relating to the current run of AutoML
+// @return {::} Value of seed is set
 runModels.setSeed:{[config]
   system"S ",string config`seed;
   }
 
 // @kind function
 // @category runModels
-// @fileoverview Apply TTS to keep holdout for feature impact plot and testing
+// @desc Apply TTS to keep holdout for feature impact plot and testing
 //   of best vanilla model
-// @param config {dict} Information relating to the current run of AutoML
-// @param tts {dict} Feature and target data split into training/testing sets
-// @return {dict} Training and holdout split of data
+// @param config {dictionary} Information relating to the current run of AutoML
+// @param tts {dictionary} Feature and target data split into training/testing
+//   sets
+// @return {dictionary} Training and holdout split of data
 runModels.holdoutSplit:{[config;tts]
   ttsFunc:utils.qpyFuncSearch config`trainTestSplit;
   ttsFunc[tts`xtrain;tts`ytrain;config`holdoutSize]
   }
 
 // @kind function
 // @category runModels
-// @fileoverview Seeded cross validation function, designed to ensure that 
+// @desc Seeded cross validation function, designed to ensure that 
 //   models will be consistent from run to run in order to accurately assess
 //   the benefit of updates to parameters.
-// @param tts {dict} Feature and target data split into training/testing sets
-// @param config {dict} Information relating to the current run of AutoML
-// @param modelTab {tab} Models to be applied to feature data
-// @return {(bool[];float[])} Predictions and associated actual values for each
-//   cross validation fold
+// @param tts {dictionary} Feature and target data split into training/testing
+//   sets
+// @param config {dictionary} Information relating to the current run of AutoML
+// @param modelTab {table} Models to be applied to feature data
+// @return {list} Predictions and associated actual values for
```

Comment:
This should be `(bool[]|float[])` because it's not a mixed list. Returns a list of booleans or floats